free-1.md

```diff
@@ -10,41 +10,157 @@ _What need is met by your idea? Who are you helping? What is that person's
+I TeX-ed tableau trees in the latest homework but it required a lot of trial
+and error as well as reading a _lot_ of samples of various tree structures.
+I would be really happy if there was an easier way for me to draw trees for 
+my homework.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Writing logic proofs in math.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Like LaTeX, the programmer would write text to a file which would then be
+compiled to the image of a tree.
+One possible way a program could look is
+```
+(A \or B) \and C \check
+C
+A \or B
+  A
+    branch1
+    branch2
+  B
+```
+which would produce the following tree
+
+![tree](/images/tree.png)
+
+This uses LaTeX syntax, such as backslashes to invoke certain symbols, which
+is familiar to many people who work with math.
+The structure is more easily readable for statements that stack (such as
+C and A \or B) compared to the equivalent tree in many tree packages in LaTeX
+which expects a statement on the next line to be the child of the previous
+statement (so A \or B would be the child of C) which would then draw a line
+between them. 
+This would go against tableau convention.
+The package prooftrees solves this issue but still requires A \or B to be
+indented relative to C, which is harder to read.
+
+For logicians, there should not be a need to specify math mode marked by \$
+since everything should be mathematical symbols.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would read the entire file to calculate how much branching is
+requested. 
+It would also calculate how long each statement in the tree is based on how
+many symbols are used.
+This is then used to calculate the angle and length of lines in the tree for
+branching to give each statement enough space.
+
+If the program runs with no issues, it would then produces a new file, an image 
+or pdf, of the resulting tree.
+Else, it would produce a file containing error messages.
+
+Possible errors could include unrecognized commands for symbols.
+With the syntax described before, blank lines could be an error or a warning.
+An error could arise if the user tries to branch in more than two directions,
+this would throw off the distance calculation.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to draw binary trees with logic symbols.
+
+It should be possible, but maybe not necessarily easy, to customize these
+symbols. 
+
+It should be impossible to draw trees that are not binary.
+Drawing shapes other than trees is also possible after a lot of difficulty.
```

Comment:
I am curious what customizations you have in mind for the symbols and how difficult they will be to implement but other than that I think these answers really show how this is a DSL and a good one. I am curious what other shapes are possible to be made. Stick figures anyone??